### ISSUE TITLE
Error on authenticating with aws servers via pem

### ISSUE THREAD

**Issue by ruskid** [deployphp/deployer#439]
My deploy.php
```php
require 'vendor/deployer/deployer/recipe/yii.php';
set('repository', 'git@bitbucket.org:xxx.git');

server('qa', 'xxx.eu-west-1.compute.amazonaws.com')
    ->user('ubuntu')
        ->stage('staging')
    ->pemFile('~/xx/xxx.pem');
```
Error i get:
dep deploy staging
Executing task deploy:prepare
PHP Notice:  Cannot connect to xxx.compute.amazonaws.com:22. Error 110. Connection timed out in phar:///usr/local/bin/dep/vendor/phpseclib/phpseclib/phpseclib/Net/SSH2.php on line 1011

I did try to download deployer and phpseclib via composer. Getting same error.

**Comment by oanhnn** [comment]
What is Deployer version?
Please try with last version (3.0.9).

**Comment by elfet** [comment]
Yes, should be fixed in 3.0.9.
